### PR TITLE
Enable user to search for an activity, redemption e.t.c

### PR DESCRIPTION
**Pull request by Chris-Maina**
##### What does this PR do?
Enables the user to search a text using the search input

##### Description of Task to be completed?
 - add Navbar folder
 - move NavbarContainer and tests to Navbar folder
 - add search utility function
 - add search action
 - add reducer to handle SEARCH action
 - add search on all the pages
 - add onClick on search icon in search input 

##### Screenshots?
Here is a link with a [demo](https://www.loom.com/share/1debe5a0de6a4a2db29cb8577caf768e)

### CODE REVIEW

File: src/app/ApproveActivities/components/ApproveActivitiesComponent.js [hunk 1]
```diff
@@ -18,12 +18,12 @@ const ApproveActivitiesComponent = ({ activities }) => {
   } else {
     tableBodyHtml = activities.map((activity) => {
       const {
-        id, activityDate, points, description, category,
+        id, createdAt, points, description, category,
```
Comment (alexmochu) [inline src/app/ApproveActivities/components/ApproveActivitiesComponent.js]: @Chris-Maina You should revert this change since we only use `createdAt` when fetching data from apiary.

Comment (Chris-Maina) [reply src/app/ApproveActivities/components/ApproveActivitiesComponent.js]: Yes we can. I just noticed apiary did not have activityDate and thought backend had made changes.

Comment (alexmochu) [reply src/app/ApproveActivities/components/ApproveActivitiesComponent.js]: Backend is still using activityDate, so if this is merged it might cause a bug when visiting the verify activities page. In this case, we should update Apiary to use activityDate which is much easier.
